### PR TITLE
2.43 uk update

### PR DESCRIPTION
**Pull request by arkid15r**
A draft PR to be submitted to @git-l10n later

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -1801,25 +1799,22 @@ msgstr ""
...
+msgstr "не вдалося прочитати індекс"
 
 msgid "editing patch failed"
 msgstr "не вдалося відредагувати латку"
 
 #, c-format
-msgid "Could not stat '%s'"
-msgstr "Не вдалося виконати stat \"%s\""
+msgid "could not stat '%s'"
+msgstr "не вдалося виконати stat '%s'"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "не вдалося виконати stat \"%s\""
```

File: po/uk.po [hunk 2]
```diff
@@ -17461,7 +17476,14 @@ msgid "failed to read the cache"
...
 msgid "multi-pack-index OID fanout is of the wrong size"
-msgstr "multi-pack-index OID fanout має невірний розмір"
+msgstr "multi-pack-index OID розсіювання має невірний розмір"
+
+msgid "multi-pack-index OID lookup chunk is the wrong size"
+msgstr "multi-pack-index шматок пошуку OID має невірний розмір"
+
+msgid "multi-pack-index object offset chunk is the wrong size"
+msgstr ""
+"multi-pack-index необхідній шматок зміщення об'єкта має невірний розмір"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"multi-pack-index необхідній шматок зміщення обʼєкта має невірний розмір"
```

**Review by kasya — COMMENTED** [review]
👍🏼

File: po/uk.po (hunk 2 above)
Comment (arkid15r) [reply po/uk.po]: 💯

**Review by kasya — COMMENTED** [review]
🏆

**Comment by arkid15r** [comment]
Closing this and filing target repo PR based on this one.

**Pull request closed without merging**